examples/training/classification/imagenet/basic_training.py

```diff
@@ -85,21 +85,41 @@
     "Keyword argument dictionary to pass to the constructor of the model being trained",
 )
 
+flags.DEFINE_string(
+    "learning_rate_schedule",
+    "ReduceOnPlateau",
+    "String denoting the type of learning rate schedule to be used",
+)
+
+flags.DEFINE_integer(
+    "warmup_steps_percentage",
+    10,
+    "For how many steps expressed in percentage of total steps should the schedule warm up if we're using the warmup schedule",
+)
+
+flags.DEFINE_integer(
+    "warmup_hold_steps_percentage",
+    10,
```

Comment:
is there any particular reason that this is the same as default warmup_steps_percentage?

Reply:
Other than it worked for me empirically before - no. I've set it as the default to fall back to, with the intention of the user manually setting it to any number between `0..total_steps`